src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/LongPollingTransport.cs

```diff
@@ -26,15 +26,70 @@ public LongPollingTransport(CancellationToken timeoutToken, PipeReader applicati
             _logger = loggerFactory.CreateLogger<LongPollingTransport>();
         }
 
+        public async Task ProcessFirstRequestAsync(HttpContext context, CancellationToken token)
+        {
+            if (!token.IsCancellationRequested)
```

Comment:
Why are we checking the token here?

Reply:
If we do decide to keep it, here's a style thing.

if (token.IsCancellationRequested)
{
    return;
}

Reply:
I thought we needed it for one of the test edge cases but turns out I can remove it from the method completely